src/app/[locale]/servicios/[servicios]/page.tsx

```
@@ -1,53 +1,211 @@
 import Text from "antd/lib/typography/Text";
 import BasicSection from "@/components/BasicSection/BasicSection";
 import { data, DataService, Service } from "@/utils/data";
 import { useTranslations } from "next-intl";
+import Image from "next/image";
+import Head from "next/head";
+import { getEmoji } from "@/utils/emojiFunction";
+import Link from "next/link";
+import { ExclamationCircleOutlined } from "@ant-design/icons";
+import styles from "./servicios.module.css";
 
 const Service = ({ params }: { params: { servicios: string } }) => {
```

Biome (1.9.4)

[error] Shouldn't redeclare 'Service'. Consider to delete it or rename it.

'Service' is defined here:


(lint/suspicious/noRedeclare)